beta/index.md

```diff
@@ -0,0 +1,119 @@
+# Gravwell 4.0 Big Bang Beta
+
+Greetings and salutations!
+
+You are reading this message because you are in the early access group for Gravwell 4.0 Beta, the Big Bang release! We sincerely thank you for your participation and look forward to feedback and bug reports.
+
+## Current "State of the Beta"
+
+### Desired Testing
+
+Testing desires for this sprint (in order of priority)
+
+* Browsing, installing, and managing kits
+* Search template creation and use to create "investigative" dashboards
+* Playbooks - New features are coming to enrich playbooks a bit more, but testing of creating and using them as markdown READMEs is the desire for this phase
+* Macros
+* Templates
+
+
+We are actively iterating on these features and they may change entirely so maybe don't put a ton of work in building stuff around them. :)
+
+* Building Kits - we intend to make it possible for the community to develop and publish kits but for now we are focused on testing Gravwell published and signed kits
+* Actionables - We are playing with how actionables work and including the idea of "active" and "passive" triggers that happen when things are regex-able vs only available when highlighted.
+* Search Studio
+* Search templates need an optional "prefix" and "postfix" in order to support optional variables.
+
+
+### Known Issues
+
+Here are the issues we are aware of, so you don't necessarily need to test or report them
+
+* Long-running chrome tabs can become unresponsive. We believe we've got an angle on this but be advised that you may need to close the Gravwell UI and open in a new tab or otherwise refresh your page in order to resolve this issue. If you can consistently reproduce this, please jot down your path and pop that over to beta@gravwell.io
```

Comment:
```suggestion
* Long-running chrome tabs can become unresponsive. We believe we've got an angle on this but be advised that you may need to close the Gravwell UI and open in a new tab or otherwise refresh your page in order to resolve this issue. If you can consistently reproduce this, please jot down your path and pop that over to beta@gravwell.io.
```